pkgs/applications/misc/hovercraft/default.nix

```diff
@@ -0,0 +1,35 @@
+{ lib
+, buildPythonApplication
+, isPy3k
+, fetchFromGitHub
+, manuel
+, setuptools
+, docutils
+, lxml
+, svg-path
+, pygments
+, watchdog
+}:
+
+buildPythonApplication rec {
+  pname = "hovercraft";
+  version = "2.6";
+  disabled = ! isPy3k;
+
+  src = fetchFromGitHub {
```

Comment:
I'm assuming here that the tests weren't distributed at `pypi`?

Reply:
yes: https://github.com/regebro/hovercraft/blob/master/setup.py#L42

Reply:
exactly, i was using pypi before